### ISSUE TITLE
notification not appear in anroid 13

### ISSUE THREAD

**Issue by m-a-wahab** [Yukams/background_locator_fixed#75]
notification not appear in anroid 13

**Comment by dtkdt100** [comment]
same here, I use [flutter local notification](https://pub.dev/packages/flutter_local_notifications) for now

**Comment by mziadd** [comment]
Is this library works with android 13 ?

**Comment by dtkdt100** [comment]
Yes. Bus there is a race condition (#77 ), so I would recommend to use the solutions in the answer until @Yukams will fix it properly

**Comment by mziadd** [comment]
Thank you for replay

**Comment by mziadd** [comment]
@dtkdt100 @m-a-wahab 

i found a workaround to fix it, 
using `flutter_local_notification`s library
in main at start of the application
add this line 
 ```
await FlutterLocalNotificationsPlugin()
        .resolvePlatformSpecificImplementation<AndroidFlutterLocalNotificationsPlugin>()
        .requestPermission();
```

make sure   `WidgetsFlutterBinding.ensureInitialized();` is enabled

**Comment by dtkdt100** [comment]
Thanks @mohammedX6 !!
